lib/cc/engine/bundler_audit/result.rb

```diff
@@ -0,0 +1,85 @@
+module CC
+  module Engine
+    module BundlerAudit
+      class Result
+        GEM_REGEX = /^\s*(?<name>\S+) \([\d.]+\)/.freeze
+        SEVERITIES = {
+          high: "critical",
+          medium: "normal",
+          low: "info",
+        }.freeze
+
+        def initialize(result, gemfile_lock)
+          @gem = result.gem
+          @advisory = result.advisory
+          @gemfile_lock = gemfile_lock
+        end
+
+        def to_issue
+          {
+            categories: ["Security"],
```

Comment:
I was planning to update the rubocop config with our styleguide in a separate PR and address any CC issues that come up as a result. Sound good or should I do that in this PR?